### DISCUSSION TITLE
Integration with POSCON

### DISCUSSION THREAD

**Discussion by fisicorj** [albar965/littlenavmap#872] [Ideas]
It would be great if you could add the option to monitor the POSCON network, in the same way as Vatsim and IVAO.

At POSCON there are APIs available, but there is currently no support in the app.

**Comment by albar965** [comment]
Certainly makes a lot of sense. I added an issue and scheduled this for 3.0: #873
I cannot do this for the next 2.8. Otherwise it will never be finished, sorry. The API looks fairly different to what VATSIM and IVAO use.

**Comment by fisicorj** [comment]
Good Morning. Here in Brazil it is still daylight. haha

No problem, it was a suggestion, no need to rush. The work you do with
Little navmap is amazing.

Thanks for the feedback.

Em ter., 22 de mar. de 2022 às 09:22, Alexander Barthel <
***@***.***> escreveu:

> Certainly makes a lot of sense. I added an issue and scheduled this for
> 3.0: #873 <https://github.com/albar965/littlenavmap/issues/873>
> I cannot do this for the next 2.8. Otherwise it will never be finished,
> sorry. The API looks fairly different to what VATSIM and IVAO use.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/albar965/littlenavmap/discussions/872#discussioncomment-2414377>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AFTEBAUXHCQK2N2GUOI5RQLVBG3O7ANCNFSM5RIJWZTA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 

*Att, *
*Manoel Guilherme F. Moraes *Tel: +55 21 982-878-726
@fisicorj -  me siga no Twitter
